src/components/GeoPointEditor/GeoPointEditor.react.js

```diff
@@ -33,6 +33,17 @@ export default class GeoPointEditor extends React.Component {
     this.refs.longitude.addEventListener('keypress', this.handleKeyLongitude);
   }
 
+  componentWillReceiveProps(props) {
```

Comment:
How is this the GeoPointEditor change related to this PR?

Reply:
It is not. Since we were using this geopoint editor on EditRow dialog, we have noticed a bug here.
So I just fixed along the way.

Reply:
Can you please revert this and create a separate PR for each change that is not part of this feature? With a description of what the PR fixes. Just for our future selves to made debugging easier :)

Reply:
Sure can and will do. Also found some other minor bugs which will be fixed in that new PR.

Reply:
Thanks, great job fixing some bugs along the way 😃